server/sonar-server/src/main/java/org/sonar/server/es/NewIndex.java

```diff
@@ -194,13 +271,14 @@ public Object getProperty(String key) {
   /**
    * Helper to define a string field in mapping of index type
    */
-  public static class StringFieldBuilder {
+  public static abstract class StringFieldBuilder {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AModifiersOrderCheck)